src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,4 @@
+NuGet.Commands.TransitiveNoWarnUtils.NodeWarningProperties.ExtractPathNoWarnProperties(string libraryId) -> System.Collections.Generic.HashSet<NuGet.Common.NuGetLogCode>
+NuGet.Commands.TransitiveNoWarnUtils.NodeWarningProperties.Merge(NuGet.Commands.TransitiveNoWarnUtils.NodeWarningProperties other) -> NuGet.Commands.TransitiveNoWarnUtils.NodeWarningProperties
+static NuGet.Commands.TransitiveNoWarnUtils.NodeWarningProperties.Create(System.Collections.Generic.HashSet<NuGet.Common.NuGetLogCode> projectWide, System.Collections.Generic.Dictionary<string, System.Collections.Generic.HashSet<NuGet.Common.NuGetLogCode>> packageSpecific) -> NuGet.Commands.TransitiveNoWarnUtils.NodeWarningProperties
+static readonly NuGet.Commands.TransitiveNoWarnUtils.NodeWarningProperties.Empty -> NuGet.Commands.TransitiveNoWarnUtils.NodeWarningProperties
```

Comment:
All of these APIs appear to be called from within the same assembly. Can we make all these APIs internal?

Customers using NuGet.Commands in their own app probably don't have a reason to use these APIs.

Reply:
Sure.  Going forward, it would make sense to make most of the APIs in this file internal.  Maybe everything except the CreateTransitiveWarningPropertiesCollection call.  I understand that would require a major version change.